package/chalicelib/reportGenerator.py

```diff
@@ -284,40 +339,70 @@ def create_report_body(self, user, response_by_account):
 
         return report
 
-    def send_management_report(self, recipients):
+    def send_management_report(self, recipient, clean=True):
         """
         Email a report, tailored to managers, to a list of recipients.
 
         :param list(str) recipients: The recipients of the email. Defaults to the value of users.
         """
+        if not os.path.exists("/tmp/"): # create directory to store graphs in
+            os.mkdir("/tmp/")
+
+        if not os.path.exists("/tmp/%s" % recipient):
+            os.mkdir("/tmp/%s" % recipient)
+
         response_by_account = dict()
 
         # Determine expenditures across all accounts.
         for acct_num in self.account_nums:
-            response = self.api_call(account_nums=[acct_num])
-            processed = self.process_api_response(response)
-            response_by_account[acct_num] = processed
 
-        report = self.create_management_report_body(response_by_account)
+            response_by_account[acct_num] = {}
+            for category in ['Owner', 'Service']:  # Create a separate report grouped by each of these categories
+                response = self.api_call(account_nums=[acct_num], group_by=category)
+                processed = self.process_api_response(response, levels=1)
+                response_by_account[acct_num][category] = processed
+
+        # Create graphics.
+        for acct in response_by_account:
+            # Make a graph for this account organized by owner and save it as a png
```

Comment:
This can probably be abstracted into another function too.

Reply:
done